.env.production

```diff
@@ -6,4 +6,4 @@ PUBLIC_LIQUID_EXPLORER=https://liquid.network
 PUBLIC_DOMAIN=coinos.io
 PUBLIC_COINOS_PUBKEY=ba80990666ef0b6f4ba5059347beb13242921e54669e680064ca755256a1e3a6
 PUBLIC_COINOS_RELAY=wss://relay.coinos.io
-PUBLIC_COINOS_NETWORK=bitcoin
+PUBLIC_COINOS_NETWORK=regtest
```

Comment:
This should stay bitcoin for .env.production but we need to add regtest to .env.sample